LICENSE

```diff
@@ -310,7 +307,10 @@
     python/mxnet/onnx/mx2onnx/_export_onnx.py
     python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py
     python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py
-
+    src/operator/numpy/np_einsum_op-inl.h
+    src/operator/numpy/np_einsum_op.cc
+    src/operator/numpy/np_einsum_path_op-inl.h
+    
```

Comment:
I already took care of moving these files here: https://github.com/apache/incubator-mxnet/pull/20492

Reply:
okay, in thse case, I will just remove these 3 lines